app/results.py

```diff
@@ -51,6 +51,19 @@ def is_possible_phone_number(query_str):
     return re.search(POSSIBLE_PHONE_NUMBER_RE,
         unicodedata.normalize('NFKC', unicode(query_str)))
 
+def max_word_length(query_words):
+    max_length = max(map(len, query_words))
+    # Count series of CJK characters as one word.
```

Comment:
Are you doing this because each CJK character is treated as one word in query_words even if it originally doesn't have whitespace e.g., "山田" -> ["山", "田"]? Have you verified that behavior? If so, can you explain it in the comment here?